src/components/OfferCard/OfferCard.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { View } from 'react-native';
+import styled from 'styled-components/native';
+import { CachedImage } from 'react-native-cached-image';
+
+// components
+import ShadowedCard from 'components/ShadowedCard';
+import { BaseText } from 'components/Typography';
+import Button from 'components/Button';
+import Spinner from 'components/Spinner';
+
+// utils
+import { fontSizes, fontStyles } from 'utils/variables';
+import { themedColors } from 'utils/themes';
+
+type ImageObject = {
+  uri: string,
+}
+
+type ButtonProps = {
+  label?: string,
+  onPress?: () => void,
+  isDisabled?: boolean,
+  isLoading?: boolean,
+  isSecondary?: boolean,
+}
+
+type Props = {
+  isDisabled?: boolean,
+  onPress: () => void,
+  labelTop: string,
+  valueTop: string | number,
+  cardImageSource?: string | ImageObject,
+  cardTopButton?: ButtonProps,
+  labelBottom: string,
+  valueBottom: string | number,
+  cardMainButton?: ButtonProps,
+  cardNote?: string,
+}
+
+const CardWrapper = styled.TouchableOpacity`
+  width: 100%;
+`;
+
+const CardRow = styled.View`
+  flex: 1;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: ${props => props.alignTop ? 'flex-start' : 'flex-end'};
+  padding: 10px 0;
+  ${({ withBorder, theme }) => withBorder
+    ? `border-bottom-width: 1px;
+       border-bottom-color: ${theme.colors.border};`
+    : ''}
```

Comment:
Hm, I feel like this would look cleaner 🤔 
```suggestion
    : ''
  }
```